ResearchKit/Common/ORKAnswerFormat.m

```diff
@@ -1754,22 +1754,23 @@ - (NSNumber *)minimumNumber {
     return @(1);
 }
 - (NSNumber *)maximumNumber {
-    return @(self.textChoices.count);
+    return @(_textChoices.count);
 }
-- (NSNumber *)defaultNumber {
+- (id<NSObject, NSCopying, NSCoding>)defaultAnswer {
     if (_defaultIndex < 0 || _defaultIndex >= _textChoices.count) {
         return nil;
     }
-    return @(_defaultIndex);
+   
+    return @[[_textChoices objectAtIndex:_defaultIndex].value];
```

Comment:
After all, use `[self textChoiceForIndex:_defaultIndex]` for safety? What do you think?